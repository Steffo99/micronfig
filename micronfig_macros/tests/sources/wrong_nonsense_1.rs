micronfig::config! {
	garasauto!"68"3469l
}

fn main() {

}