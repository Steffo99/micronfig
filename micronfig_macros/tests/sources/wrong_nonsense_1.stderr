error: expected `,`
 --> tests/sources/wrong_nonsense_1.rs:2:11
  |
2 |     garasauto!"68"3469l
  |              ^
